.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,7 +20,8 @@ A clear and concise description of what the bug is.
 
 ### Environment
 * Android Studio version:
-* Android Gradle version:
+* Android Gradle Plugin version:
```

Comment:
I think we want Gradle and AGP (in both files):

```suggestion
* Gradle version:
* Android Gradle Plugin (AGP) version:
```

Reply:
https://github.com/bugsnag/bugsnag-android-gradle-plugin/pull/397 has been created as out tool creates new branches each time a change is made